src/session/index.ts

```diff
@@ -166,6 +167,33 @@ export default abstract class Session extends EventEmitter {
         throw new Error('Malformed service message or message handler is not implemented');
     }
 
+    get syncCookies () {
+        const parsedSyncCookies = [] as string[];
+
+        while (true) {
```

Comment:
Infinite loop is wrong solution.

Just use `this.cookies.syncCookies.map()` to generate sync cookies.